samples/katib/early-stopping.ipynb

```diff
@@ -29,212 +29,16 @@
-      "Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.6/dist-packages (from importlib-metadata->jsonschema>=3.0.1->kfp==1.0.4->kfp-tekton==0.4.0) (2.1.0)\n",
-      "Requirement already satisfied: oauthlib>=3.0.0 in /usr/local/lib/python3.6/dist-packages (from requests-oauthlib->kubernetes==11.0.0->kfp-tekton==0.4.0) (3.1.0)\n",
-      "Requirement already satisfied: wheel in /usr/lib/python3/dist-packages (from strip-hints->kfp==1.0.4->kfp-tekton==0.4.0) (0.30.0)\n",
-      "Building wheels for collected packages: kfp-tekton, kfp\n",
-      "  Building wheel for kfp-tekton (setup.py) ... \u001b[?25ldone\n",
-      "\u001b[?25h  Created wheel for kfp-tekton: filename=kfp_tekton-0.4.0-py3-none-any.whl size=32814 sha256=ad561c5904811e0bc21faa694abcfd9d679b32ced94c9ab98b34105364183ade\n",
-      "  Stored in directory: /home/jovyan/.cache/pip/wheels/41/f0/6d/bfbc4f2e2321e94795a76942a6f14ed7b5d3d797d6ef9a490b\n",
-      "  Building wheel for kfp (setup.py) ... \u001b[?25ldone\n",
-      "\u001b[?25h  Created wheel for kfp: filename=kfp-1.0.4-py3-none-any.whl size=160907 sha256=34188dc0c2538baddacc37015a26a14127ea98ccddb0da3e426326c639cb0a84\n",
-      "  Stored in directory: /home/jovyan/.cache/pip/wheels/e5/a2/42/9bbf340f69f38f67e7c26434cf0e7cd68c3bd4453bf57ece5e\n",
-      "Successfully built kfp-tekton kfp\n",
-      "Installing collected packages: kubernetes, kfp, kfp-tekton\n",
-      "  Attempting uninstall: kfp\n",
-      "    Found existing installation: kfp 1.1.1\n",
-      "    Uninstalling kfp-1.1.1:\n",
-      "      Successfully uninstalled kfp-1.1.1\n",
-      "\u001b[33m  WARNING: The scripts dsl-compile and kfp are installed in '/home/jovyan/.local/bin' which is not on PATH.\n",
-      "  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.\u001b[0m\n",
-      "\u001b[33m  WARNING: The script dsl-compile-tekton is installed in '/home/jovyan/.local/bin' which is not on PATH.\n",
-      "  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.\u001b[0m\n",
-      "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
-      "kubeflow-katib 0.10.1 requires kubernetes==10.0.1, but you have kubernetes 11.0.0 which is incompatible.\u001b[0m\n",
-      "Successfully installed kfp-1.0.4 kfp-tekton-0.4.0 kubernetes-11.0.0\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "# Update the PIP version.\n",
     "!python -m pip install --upgrade pip\n",
-    "!pip install kfp==1.1.1\n",
-    "!pip install kubeflow-katib==0.10.1\n",
-    "!pip install kfp-tekton==0.4.0"
+    "!pip install kfp==1.4.0\n",
+    "!pip install kubeflow-katib==0.11.0\n",
+    "!pip install kfp-tekton==0.7.0\n",
+    "!pip install kubernetes==12.0.0"
```

Comment:
Maybe add a command to restart the kernel after this? Since kubeflow-katib may override the kubernetes client version.
```
from IPython.core.display import HTML
HTML("<script>Jupyter.notebook.kernel.restart()</script>")
```

Reply:
it already has a `restart` section below :-)

Reply:
oh sorry didn't see that, thanks